### ISSUE TITLE
corrupted glade keeps growing back by the time i encounter it again

### ISSUE THREAD

**Issue by okami465** [Kitteh6660/Corruption-of-Champions-Mod#653]
### Overview
	As the title says, since the corrupted glade's spawn rate is very low when it gets to a lower percentage, it becomes almost impossible to find. it WAS at 1% but by the time i could find another one it grew back up to 8%
	
### Expected behavior
	being able to find the corrupt glade and destroy it
	
### Actual behavior
	the corrupt glade never showing up until the percentage grows back up

### Steps to reproduce the issue
just get the corrupt glade to a very low percentage

If possible, attach a save file that reproduces the issue.
Please be aware that any files uploaded here are publicly accessible and cannot be deleted (but the link to them can be removed).

[game-file.zip](https://github.com/Kitteh6660/Corruption-of-Champions-Mod/files/1013163/game-file.zip)

**Comment by Foxwells** [comment]
Nope, this is normal. It's very hard to find because they're almost extinct. Finish them with a follower's help.

**Comment by aimozg** [comment]
Maybe it's better to make 0.2 + 0.8 x (% left)

**Comment by Stadler76** [comment]
Yeah, since there are many other encounters in the forest, the chance would be far less, than 1% to encounter the last corrupted glade, if you had destroyed 99% of them.

btw: While you're fixing this you should also replace
```as3
if (flags[kFLAGS.CORRUPTED_GLADES_DESTROYED] == 100) outputText("\n\nThat should be the last of the glades! <b>Corrupted Glades are now extinct.</b>");
```
with
```as3
if (flags[kFLAGS.CORRUPTED_GLADES_DESTROYED] >= 100) outputText("\n\nThat should be the last of the glades! <b>Corrupted Glades are now extinct.</b>");
```
in CorruptedGlade.as
and/or add
```as3
```as3
if (flags[kFLAGS.CORRUPTED_GLADES_DESTROYED] > 100) flags[kFLAGS.CORRUPTED_GLADES_DESTROYED] = 100;
```
after
```as3
flags[kFLAGS.CORRUPTED_GLADES_DESTROYED] += destroyAmount;
```
```

**Comment by Kitteh6660** [comment]
I'll make some tweaks so for every 15% progress, the time taken for the glades to recover increase by 1 day.

If the glades are close to extinction, it'll take 9 days for progress to be set back by 1%.

**Comment by Kitteh6660** [comment]
Should now be addressed in upcoming 1.4.6 release.
